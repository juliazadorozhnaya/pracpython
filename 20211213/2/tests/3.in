[399, 812, 959, 315, 189, 602, 371, 252, 616, 483, 413, 434, 567, 385, 686, 588, 952, 98, 518, 630, 672, 476, 595, 637, 462, 49, 553, 280, 196, 924, 133, 224, 161, 714, 315, 910, 364, 126, 259, 910, 42, 28, 987, 49, 301, 98, 700, 868, 637, 623, 49, 490, 364, 721, 357, 588, 42, 714, 854, 378, 574, 168, 700]
