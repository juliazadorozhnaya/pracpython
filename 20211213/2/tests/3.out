[28, 42, 42, 49, 49, 49, 98, 98, 126, 133, 161, 168, 189, 196, 224, 252, 259, 280, 301, 315, 315, 357, 364, 364, 371, 378, 385, 399, 413, 434, 462, 476, 483, 490, 518, 553, 567, 574, 588, 588, 595, 602, 616, 623, 630, 637, 637, 672, 686, 700, 700, 714, 714, 721, 812, 854, 868, 910, 910, 924, 952, 959, 987]
